### PR TITLE
Feature/store-disk-space

### PR DESCRIPTION
**Pull request by nmlinaric**
Exposed endpoint that allows to store disk space  ( total free space and taken space) and associate it with a node based on the URL and actor address provided.

### CODE REVIEW

File: src/Migrations/20191230143918-create-disk-information.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {QueryInterface, DataTypes} from 'sequelize';
+
+module.exports = {
+    up: (queryInterface: QueryInterface) => {
+        return queryInterface.createTable('DiskInformations', {
```
Comment (morrigan) [inline src/Migrations/20191230143918-create-disk-information.ts]: This is funny as there is no plural for information :) You can use `freezeTableName: true` and use it as `DiskInformation`

File: src/Services/Database.ts [hunk 2]
```diff
@@ -93,6 +94,8 @@ export class Database {
 
     private initModels(): void {
         Node.initialize(this.sequelize);
+        DiskInformation.initialize(this.sequelize);
+        Node.hasMany(DiskInformation);
```
Comment (morrigan) [inline src/Services/Database.ts]: Why is this here? You can define this in the model, same as you did DiskInformation with `belongsTo`

File: src/Migrations/20191230143918-create-disk-information.ts (hunk 1 above)
Comment (morrigan) [reply src/Migrations/20191230143918-create-disk-information.ts]: After changes, delete db data and run the migrations again and test it to make sure everything still works.

File: src/Models/DiskInformation.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export class DiskInformation extends Model {
+
+    private freeSpace: number;
+    private takenSpace: number;
+
+    public static initialize(sequelize: Sequelize) {
+        this.init({
+            freeSpace: {
+                type: DataTypes.NUMBER,
```
[the remark this answers is not in the document]
Comment (nmlinaric) [reply src/Models/DiskInformation.ts]: Something messed up while generating the models and migrations with cli, I'll correct it to be BIGINT.

File: src/Services/Database.ts (hunk 2 above)
Comment (nmlinaric) [reply src/Services/Database.ts]: I must've left it by accident while testing something, but i agree it's not supposed to be here.
